src/theme/variables.scss

```diff
@@ -8,6 +8,41 @@
 // Be sure that you only ever include this mixin once!
 @include mat-core();
 
+.mat-toolbar {
+  background: #564DFC !important;
```

Comment:
Avoid `!important` exception. I think we should use a common CSS class to set the background and color. Though this will result in duplicate code on HTML, bug it makes sure we won't need to override the rule with `!important` again when we have to. E.g. we need to change the background and color for a specific `mat-toolbar` on a specific page.